docs/api/browser-window.md

```diff
@@ -983,6 +983,29 @@ Same as `webContents.capturePage([rect, ]callback)`.
   * `httpReferrer` String - A HTTP Referrer url.
   * `userAgent` String - A user agent originating the request.
   * `extraHeaders` String - Extra headers separated by "\n"
+  * `postData` Array - An array of `upload` objects which
```

Comment:
We'll have to update https://github.com/electron/electron-docs-linter/blob/f681fa91ae43e6635ad5ed2f84db32fd4ec6a915/lib/parsers.js#L7 to support this pattern, but yeah.

Thoughts on this style, @kevinsawicki and @enlight?